client/my-sites/stats/post-trends/day.jsx

```diff
@@ -103,10 +96,10 @@ module.exports = React.createClass( {
 				<Popover context={ this.refs && this.refs.day }
 					isVisible={ this.state.showPopover }
 					position="top"
-					onClose={ noop }
 					className="chart__tooltip is-streak"
+					onClose={ noop }
```

Comment:
Is the relocation an indication that you'd tried to remove the prop altoghether? 😉  That was my first thought, though looks like this is required in `<Popover />`. Don't see any reason why we couldn't make it optional... side-concern of course.